deploy/signup-apitestrig/README.md

```
@@ -16,8 +22,18 @@ chmod 777 /srv/nfs/mosip/<sandbox>/apitestrig
 ```
 * Add the following entry to the /etc/exports file:
 ```
-/srv/nfs/mosip/<sandbox>/apitestrig *(ro,sync,no_root_squash,no_all_squash,insecure,subtree_check)
+/srv/nfs/mosip/<sandbox>/apitestrig *(rw,sync,no_root_squash,no_all_squash,insecure,subtree_check)
+```
+* Apply export command
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)